stdeb.cfg

```diff
@@ -1,6 +1,6 @@
 [DEFAULT]
-Depends: subversion, mercurial, git-core, bzr, python-yaml, python-vcstools (>= 0.1.38), python-catkin-pkg, python-rosdistro (>=0.3.0), python-wstool (>=0.1.12)
-Depends3: subversion, mercurial, git-core, bzr, python3-yaml, python3-vcstools (>= 0.1.38), python3-catkin-pkg, python3-rosdistro (>=0.3.0), python3-wstool (>=0.1.12)
+Depends: subversion, mercurial, git-core, bzr, python-yaml, python-vcstools (>= 0.1.38), python-catkin-pkg, python-rosdistro (>=0.3.0), python-wstool (>=0.1.12), rospkg
+Depends3: subversion, mercurial, git-core, bzr, python3-yaml, python3-vcstools (>= 0.1.38), python3-catkin-pkg, python3-rosdistro (>=0.3.0), python3-wstool (>=0.1.12), rospkg
```

Comment:
And `python3-rospkg`:

http://repositories.ros.org/ubuntu/main/pool/main/p/python3-rospkg/